src/openforms/emails/tests/test_tasks.py

```diff
@@ -177,3 +181,92 @@ def test_no_recipients(self):
+            TimelineLogProxyFactory.create(
+                template="logging/events/registration_failure.txt",
+                content_object=failed_submission_1,
+                extra_data={
+                    "log_event": "registration_failure",
+                    "include_in_daily_digest": True,
+                },
+            )
+
+        # 2nd failure
+        with freeze_time("2023-01-02T18:30:00+01:00"):
+            TimelineLogProxyFactory.create(
+                template="logging/events/registration_failure.txt",
+                content_object=failed_submission_2,
+                extra_data={
+                    "log_event": "registration_failure",
+                    "include_in_daily_digest": True,
+                },
+            )
+
+        # 2nd form with 1 failure in the past 24 hours
+        form_2 = FormFactory.create()
+        failed_submission = SubmissionFactory.create(
+            form=form_2, registration_status=RegistrationStatuses.failed
+        )
+
+        # failure
+        with freeze_time("2023-01-02T12:30:00+01:00"):
+            TimelineLogProxyFactory.create(
+                template="logging/events/registration_failure.txt",
+                content_object=failed_submission,
+                extra_data={
+                    "log_event": "registration_failure",
+                    "include_in_daily_digest": True,
+                },
+            )
+
+        with (
+            freeze_time("2023-01-03T01:00:00+01:00"),
+            patch(
+                "openforms.emails.tasks.GlobalConfiguration.get_solo",
+                return_value=GlobalConfiguration(
+                    recipients_email_digest=["user@example.com"]
+                ),
+            ),
+            patch("openforms.emails.tasks.send_mail_html") as patch_email,
+        ):
+            send_email_digest()
+
+            patch_email.assert_called_once()
+
+            args = patch_email.call_args.args
+            expected_content = dedent(
+                f"""
+                <p>
+                    Here is a summary of the failed procedures in the past 24 hours:
+                </p>
+                <h5>Registrations</h5>
+                <ul>
+                    <li>- {form_1.name} failed 2 times between 12:30 and 18:30.</li>
+                    <a href="/admin/submissions/submission/?form__id__exact={form_1.id}&needs_on_completion_retry__exact=1&from_time=24hAgo">
+                    Click here to see all the failed submissions for form {form_1.name} in the admin
+                    </a>
+                    <li>- {form_2.name} failed 1 times between 12:30 and 12:30.</li>
+                    <a href="/admin/submissions/submission/?form__id__exact={form_2.id}&needs_on_completion_retry__exact=1&from_time=24hAgo">
+                    Click here to see all the failed submissions for form {form_2.name} in the admin
+                    </a>
+                </ul>
+
+            """
+            ).strip()
```

Comment:
why do we need to mock the email sending? Django itself already sets up an in-memory mail backend, so you can introspect sent emails via `django.core.mail.outbox`.